src/accessibility/gridOutput.js

```diff
@@ -10,6 +10,10 @@ import p5 from '../core/main';
 
 //updates gridOutput
 p5.prototype._updateGridOutput = function(idT) {
+  // Check if the current rendering mode is WEBGL
+  if (this._renderer && this._renderer.isWEBGL) {
```

Comment:
Right now we don't have this property on renderers (although https://github.com/processing/p5.js/pull/5844 is open for it, it hasn't yet been merged.) For now I think maybe you can check if `this._renderer instanceof p5.RendererGL` to achieve the same thing.